src/System.Management.Automation/engine/CommandCompletion/CompletionAnalysis.cs

```diff
@@ -537,10 +547,10 @@ internal List<CompletionResult> GetResultHelper(CompletionContext completionCont
                                 completionContext.ReplacementLength = replacementLength = 0;
                                 result = GetResultForAttributeArgument(completionContext, ref replacementIndex, ref replacementLength);
                             }
-                            else
+                            else if (lastAst.Parent is DynamicKeywordStatementAst || lastAst.Parent is CommandExpressionAst)
                             {
-                                //
-                                // Handle auto completion for enum/dependson property of DSC resource,
+                                // 1. Handle scenarios such as 'configuration foo { File ab { Attributes ='
```

Comment:
> Handle auto completion for **enum/dependson** property of DSC resource

The example `configuration foo { File ab { Attributes =` is a completion for enum property, so it should not be marked as `1.`.

Reply:
Fixed.